web/src/components/pages/profile.jsx

```diff
@@ -162,12 +182,26 @@ export default class Profile extends React.Component {
           <label className="language-selector-label" htmlFor="language-selector">
             Add a language you want to contribute to
           </label>
-          <LanguageSelector disabled={this.props.pending}
+          <LanguageSelector disabled={pending}
                             name="language-selector"
-                            filters={this.props.languages} />
-          <button disabled={this.props.pending}
+                            filters={languages} />
+          <button disabled={pending}
                   onClick={this.onAdd} className="add-language">Add</button>
         </section>
+
+        <section>
+          <h2>Settings</h2>
+          <p>Experimental: There are two different tools with which you can review sentences. The normal tool lists 5 sentences per page
+          and has an approval and rejection button each. The Swiping tool displays one card at a time and you can swipe left
```

Comment:
Just my opinion:
```suggestion
          and has an approval and rejection button each. The Swiping tool displays one card at a time where you can swipe left
```

Reply:
Also, I would consider changing from swiping left __and__ right to left __or__ right (the same with approve/reject), and, changing either the order of `left and right` or `approve and reject` to match the order of actions with the order of decisions (current wording seems to imply to me you swipe left to approve and swipe right to reject)